plugins/llvm/llvm_disasm.cpp

```diff
@@ -223,13 +325,18 @@ class llvm_disassembler : public disassembler_interface {
     }
 
     void set_memory(memory m) {
+        #if LLVM_VERSION_MAJOR == 3 && LLVM_VERSION_MINOR >= 5
         mem = memory(m);
+        #else
+        mem.reset(new MemoryObject(m));
+        #endif
     }
 
     void step(int64_t pc) {
         mcinst.clear();
         uint64_t size = 0;
 
+        #if LLVM_VERSION_MAJOR == 3 && LLVM_VERSION_MINOR >= 6
```

Comment:
the check is a valid thing for both versions. If fact, a soon will merge the fix, that will contain this branch.